api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -111,7 +111,9 @@ FirecloudWorkspaceACLUpdateResponseList updateWorkspaceACL(
    */
   FirecloudNihStatus getNihStatus();
 
-  FirecloudNihStatus postNihCallback(FirecloudJWTWrapper wrapper);
+  // Submits a Shibboleth JWT for verification and update using the old-style Shibboleth service
+  // (via Firecloud Orchestration API).
+  void postNihCallback(FirecloudJWTWrapper wrapper);
```

Comment:
This is a somewhat opportunistic change / cleanup: the return value from postNihCallback was never used.